libs/web/design-system/src/lib/styles/theme.tsx

```diff
@@ -1,12 +1,56 @@
 import { ThemeOptions } from '@mui/material/styles';
 
+
+const colorPalette = {
+  primary: {
+    main: '#3f51b5',
+    light: '#757de8',
+    dark: '#002984',
+    contrastText: '#fff',
+  },
+  secondary: {
+    main: '#f50057',
+    light: '#ff4081',
+    dark: '#c51162',
+    contrastText: '#fff',
+  },
+  error: {
+    main: '#f44336',
+    light: '#e57373',
+    dark: '#d32f2f',
+    contrastText: '#fff',
+  },
+  warning: {
+    main: '#ff9800',
+    light: '#ffb74d',
+    dark: '#f57c00',
+    contrastText: '#fff',
+  },
+  info: {
+    main: '#2196f3',
+    light: '#64b5f6',
+    dark: '#1976d2',
+    contrastText: '#fff',
+  },
+  success: {
+    main: '#4caf50',
+    light: '#81c784',
+    dark: '#388e3c',
+    contrastText: '#fff',
+  },
```

Comment:
When you replaced with the old code you forgot the property:

```suggestion
  color: { white: 'whitehashvalue' }
```